hrm-domain/hrm-service/permission-rules/br.json

```diff
@@ -22,6 +22,7 @@
   "viewContact": [["everyone"]],
   "editContact": [["isSupervisor"]],
   "viewExternalTranscript": [["isSupervisor"]],
+  "viewRecording": [],
```

Comment:
@janorivera I'm not adding the actual permissions to the helplines. Can you take care on checking which permissions should be set for them? Or should I copy the `viewExternalTranscript` one?

Reply:
@GPaoloni OK I think CL and CA uses calls and probably ZW (?)
In any case I've asked Sinekhaya and based on what I know from CL I think is safe to assume that **viewRecording** can have the same value as **viewExternalTranscript**.  @dee-luo is not here to confirm about CA but I'm assuming that is only for supervisors.